lib/charms/operator_libs_linux/v2/snap.py

```diff
@@ -530,6 +537,7 @@ def ensure(
         self,
         state: SnapState,
         classic: Optional[bool] = False,
+        devmode: Optional[bool] = False,
```

Comment:
Same here, I think the type should be just `devmode: bool = False` (non-optional).

I know that makes it different from the type for `classic`. I think we could get away with changing that type annotation to just `bool` too, to match -- no one should be passing None here.

Reply:
Done